### ISSUE TITLE
Material Style in DatePicker

### ISSUE THREAD

**Issue by BenjaminHofstetter** [oblique-bit/oblique#125]
### Oblique version

11

### Browser & version

Chrome  122.0.6261.112 (Official Build) (arm64)

### Node version

v20.11.0

### Device and operating system

MacOS 14.3.1

### Is this a regression?

No

### Screenshots

<img width="286" alt="Screenshot 2024-03-12 at 10 36 50" src="https://github.com/oblique-bit/oblique/assets/8033981/b510ff4e-4c15-4237-bc2d-3f0baac860e9">

### Description

```true
I have a message on the console.
core.mjs:200 Could not find Angular Material core theme. Most Material components may not work as expected. For more info refer to the theming guide: https://material.angular.io/guide/theming

I see a bit purple in the calendar component. This color doesn't seem right. Do I have to configure a Material Theme or is this an oblique issue?
```

### Reproduction

```true
You have the same here https://oblique.bit.admin.ch/components/datepicker/examples
```

### Expected behaviour

```true
A color that matches the CI/CD of admin.ch
```

**Comment by nina-egger** [comment]
@BenjaminHofstetter 
Console Message: We received similar reports and are already looking into it. So far it doesn't seem like it's breaking anything. 

Purple border: That focus color is correct and not a bug.

**Comment by BenjaminHofstetter** [comment]
Thank you for clarification.

**Comment by nina-egger** [comment]
Note: Material theme warning is being tracked on task OUI-2934

**Comment by nina-egger** [comment]
OUI-2934 has been fixed with the version 11.1.0.
